flows/actions/testdata/send_msg.json

```diff
@@ -21,6 +21,31 @@
         },
         "read_error": "field 'topic' is not a valid message topic"
     },
+    {
```

Comment:
this tests are just to make sure the validation works but no easy way to test the validation with omitempty